# SPDX-License-Identifier: GPL-3.0
# Patch against upstream OpenSplat (https://github.com/pierotofy/OpenSplat, GPL-3.0); derivative, same license.
diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -214,4 +214,5 @@ if((GPU_RUNTIME STREQUAL "CUDA") OR (GPU_RUNTIME STREQUAL "HIP"))
     else(GPU_RUNTIME STREQUAL "HIP")
         set(GPU_INCLUDE_DIRS "${ROCM_ROOT}/include")
         target_compile_definitions(gsplat PRIVATE USE_HIP __HIP_PLATFORM_AMD__)
+        target_compile_options(gsplat PRIVATE "-I${ROCM_ROOT}/include")
     endif()
